emperor/support_files/js/controller.js

```diff
@@ -262,12 +263,54 @@ define([
       selector: '#' + scope.$divId.attr('id') + ' .emperor-plot-wrapper',
       trigger: 'none',
       items: {
+        'saveState': {
+          name: 'Save current settings',
+          icon: 'edit',
+          callback: function(key, opts) {
+            scope.saveConfig();
+          }
+        },
+        'loadState': {
+          name: 'Load saved settings',
+          icon: 'paste',
+          callback: function(key, opts) {
+            if (!FileReader) {
+              alert("Your browser does not support file loading. Please upgrade your browser.");
```

Comment:
Do we know which browsers we should recommend? In my experience it's easier to recommend a browser than to recommend users to upgrade their current version, just to have them find out the new version doesn't support the framework 😞 So I think we should just recommend that they use Google Chrome.